DistributionPackages/Neos.Presentation/Resources/Private/Fusion/Molecule/PostGrid/PostGrid.Teaser.fusion

```diff
@@ -38,21 +39,25 @@ prototype(Neos.Presentation:PostGrid.Teaser) < prototype(Neos.Fusion:Component)
                             'sm:text-4xl': props.isFeature
                         })}>{props.title}</h2>
 
-                        <p class="text-white text-sm mb-8">{props.date}</p>
+                        <time datetime={props.date && Date.format(props.date, 'Y-m-d')} class="text-white text-sm mb-8">{props.date && Date.format(props.date, 'd.m.Y')}</time>
```

Comment:
```suggestion
                        <time @if={Type.instance(props.date, '\DateTimeInterface')} datetime={Date.format(props.date, 'Y-m-d')} class="text-white text-sm mb-8">{Date.format(props.date, 'd.m.Y')}</time>
```

The prop types just make sure on development context that nothing wrong will added. We have still to make sure that everything works if someone passes a wrong input